pkgs/servers/home-assistant/custom-components/bodymiscale/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+
+  cachetools,
+}:
+
+let
+  cachetools' = cachetools.overrideAttrs (oldAttrs: rec {
```

Comment:
nixpkgs has 5.3.2, manifest has the strict constraint `cachetools==5.3.0`. Dunno if we prefer to override the package here, or patch the manifest?

Reply:
Strong preference for `pythonRelaxDeps`, since overriding of packages cannot easily be done for just this one package in the complete home-assistant context.

Reply:
Ah right, components have a dedicated manifest checker. My bad.

Reply:
Yeah, seems like some components patch it via `substituteInPlace`, others turn off the manifest check wholesale. I updated the PR to use the former option.